packages/client/src/pages/404.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import Button from '../components/Button'
+import { H1, Paragraph } from '../components/Typography'
+import notFound from '../images/svg/notFound.svg'
+
+const Wrapper = styled.div`
+  padding: 0 19px;
+  display: flex;
+  overflow: hidden;
+  height: 87vh;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const Image = styled.img`
+  margin: 0 auto;
+`
+const Container = styled.div`
+  margin-top: 8rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
These two are used in `Confirmation.tsx` as well. So instead of having duplicates you could move out thes to `layout.ts`